plugins/com.google.cloud.tools.eclipse.preferences/schema/com.google.cloud.tools.eclipse.preferences.areas.exsd

```diff
@@ -0,0 +1,178 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!-- Schema file written by PDE -->
+<schema targetNamespace="com.google.cloud.tools.eclipse.preferences" xmlns="http://www.w3.org/2001/XMLSchema">
+<annotation>
+      <appinfo>
+         <meta.schema plugin="com.google.cloud.tools.eclipse.preferences" id="com.google.cloud.tools.eclipse.preferences.areas" name="Preference Areas"/>
+      </appinfo>
+      <documentation>
+         *** EXPERIMENTAL ***
+
+Provides a mini framework for hosting conceptually-related but independently-developed preferences areas on a single page.
+      </documentation>
+   </annotation>
+
+   <element name="extension">
+      <annotation>
+         <appinfo>
+            <meta.element />
+         </appinfo>
+      </annotation>
+      <complexType>
+         <choice>
+            <element ref="area" minOccurs="1" maxOccurs="unbounded"/>
+         </choice>
+         <attribute name="point" type="string" use="required">
+            <annotation>
+               <documentation>
+                  Specifies areas extension point
+               </documentation>
+            </annotation>
+         </attribute>
+         <attribute name="id" type="string">
+            <annotation>
+               <documentation>
+                  An optional identifier; not used by preference areas.
+               </documentation>
+            </annotation>
+         </attribute>
+         <attribute name="name" type="string">
+            <annotation>
+               <documentation>
+                  An optional name; not used by preference areas.
+               </documentation>
+               <appinfo>
+                  <meta.attribute translatable="true"/>
```

Comment:
I can't find the docs for these meta. elements with a little googling, but it looks like someone tried to reinvent namespaces